test/e2e/suites/base/suite_test.go

```diff
@@ -211,6 +211,9 @@ var _ = ginkgo.SynchronizedAfterSuite(func() {
 	// It will not return error even if there is no such namespace in there that may happen in case setup failed.
 	err := cleanupTestNamespace(testNamespace, kubeClient)
 	gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+
```

Comment:
Because of the comment, this and the above content should belong to the same code block, we can delete the blank line.